agent/bench-scripts/pbench-linpack

```diff
@@ -113,24 +117,42 @@ while true; do
 done
 verify_common_bench_script_options ${tool_group} ${sysinfo}
 
-ver="$(pbench-config version ${benchmark})"
-if [[ -z "${ver}" ]]; then
-	error_log "${script_name}: package version is missing in config file"
-	exit 1
-fi
-if [[ -z "${linpack_dir}" ]]; then
-	linpack_dir="/usr/local/${script_name}-${ver}/benchmarks/linpack"
-	linpack_dir_kind="default"
-else
-	linpack_dir_kind="provided"
+function pre_check {
+	${pbench_bin}/bench-scripts/driver/linpack --pre-check ${ver}
+	return ${?}
+}
+
+if [[ ${do_pre_check} -ne 0 ]]; then
+	# We have been invoked remotely to check for the expected version of
+	# linpack installed.
+	pre_check ${@}
```

Comment:
What is in `${@}` at this point?  (I think it's just the non-switch command line arguments....)  There's nothing in the help text which mentions command line arguments.

Reply:
It won't ever be described in the help text, this is an internal hand shake of invoking the `pbench-linpack` remotely with `--pre-check` to cause the same behavior to be taken on those hosts.